middle.mkd

```diff
@@ -285,7 +288,7 @@ processing software SHALL NOT be expected to have access to coordinate
 reference systems databases.  Applications requiring CRS other than the
 default MUST assume all responsibility for reference system and
 coordinate accuracy. Furthermore, GeoJSON coordinates MUST NOT under any
-circumstances use latitude, longitude order. See Section 6,
+circumstances use latitude, longitude order. See Section 8,
```

Comment:
I note that symbolic references are possible with other toolchains.  This manual process is brittle.